src/gui/Src/Gui/BreakpointsView.cpp

```diff
@@ -146,14 +163,14 @@ void BreakpointsView::reloadData()
     {
         QString addr_text = QString("%1").arg(wBPList.bp[wI].addr, sizeof(dsint) * 2, 16, QChar('0')).toUpper();
         mSoftBPTable->setCellContent(wI, 0, addr_text);
-        mSoftBPTable->setCellContent(wI, 1, QString(wBPList.bp[wI].name));
+        mSoftBPTable->setCellContent(wI, 1, QString::fromUtf8(wBPList.bp[wI].name));

```

Comment:
Actually you don't have to use `fromUtf8` since that's the default option in Qt5 and in Qt4 we use `QTextCodec::setCodecForCStrings()`